packages/slate/src/changes/with-schema.js

```diff
@@ -146,11 +139,14 @@ function normalizeNode(change, node, schema) {
     if (!normalize) return
 
     // Run the `normalize` function to fix the node.
+    let nPath = c.value.document.getPath(n.key)
     normalize(c)
 
     // Re-find the node reference, in case it was updated. If the node no longer
     // exists, we're done for this branch.
-    n = refindNode(c, n)
+    const cacheNAndPath = refindNode(c, n, nPath)
```

Comment:
Can you change this variable to `result`?

Reply:
Yes, I will